src/main/java/eu/europa/ted/eforms/sdk/component/SdkComponentFactory.java

```diff
@@ -78,6 +80,20 @@ private void populateComponents() {
         .map(Package::getName)
         .toArray(String[]::new);
 
+    if (logger.isDebugEnabled()) {
```

Comment:
This allowed me to find the problems and can help others in the future or at least give an idea of what to look for.
The logs are easy to read (one item per line).
Not sure if we want to keep the logs as is but I would find it useful in case I have to debug this again...

Reply:
The logs would only show package names that match specific pattern (eu...). So this is not useful for others.
If logging the package names is useful, I would be OK to have it for all packages, but only at the "trace" level.

Reply:
Chnage OK